docs/UserGuide.md

```diff
@@ -107,12 +107,12 @@ Examples:
 
 Creates a new event with accompanying details (name, location, date, tag).
 
-Format: `createEvent n/EVENT_NAME l/LOCATION d/DATE [t/TAG]`
+Format: `createEvent n/EVENT_NAME l/LOCATION d/DATE [t/TAG]...`
 
 * Acceptable values for `DATE`: Dates in the format of `YYYY-MM-DD`.
 
 Examples:
-* `createEvent n/Party l/Raffles Hall d/16-09-2023` creates an event `Party` in CCACommander.
+* `createEvent n/Party l/Raffles Hall d/2023-09-16` creates an event `Party` in CCACommander.
```

Comment:
We should update the date format in the Command Summary section to YYYY-MM-DD too. Currently it is still in the DD-MM-YYYY format.

Reply:
Updated